src/deadline/job_attachments/_utils.py

```diff
@@ -94,13 +93,16 @@ def _is_relative_to(path1: Union[Path, str], path2: Union[Path, str]) -> bool:
 
 
 def _is_windows_file_path_limit() -> bool:
```

Comment:
nit: Not from this PR, but this name appears to be logically reversed from the function it calls. A name like `_windows_are_long_paths_enabled`, matching the Windows API call very closely, could be more clear.